drivers/usb/device/usb_dc_stm32.c

```diff
@@ -389,11 +389,9 @@ static int usb_dc_stm32_phy_specific_clock_enable(const struct device *const clk
 			return -ENOTSUP;
 		}
 	}
-
+#endif
```

Comment:
This fixup should be squash in commit "drivers: usb: stm32 device stack supporting the stm32h7rs" (if still applicable)